ConnectorArchicad/ConnectorArchicad/Communication/AsyncCommandProcessor.cs

```diff
@@ -27,7 +27,15 @@ internal class AsyncCommandProcessor
     {
       return Task.Run(command.Execute, token);
     }
-    catch (Exception e)
+    catch (ArgumentNullException)
```

Comment:
I think this one is only going to throw if `command.Execute` is null, which can't be the case since it's part of the ICommand interface. 

I think we don't really need to catch this as it would really be an unexpected event (provided input was invalid somehow)